xml/core_registers.xml

```diff
@@ -1,6 +1,16 @@
 <registers name="Core Debug Registers" prefix="CSR_">
     These registers are only accessible from Debug Mode.
     <register name="Debug Control and Status" short="dcsr" address="0x7b0">
+        \begin{commentary}
+            \Fintdisable is intended to be used in conjunction with \Fstep.
```

Comment:
Trying to keep the contents of the cell small. Not sure what happens if you try to put the commentary in one, but it doesn't feel right.